src/plugins/data/public/query/query_string/language_service/lib/query_result.tsx

```diff
@@ -22,10 +22,9 @@ export interface QueryStatus {
   status: ResultStatus;
   body?: {
     error?: {
-      reason?: string;
-      details: string;
+      statusCode?: number;
```

Comment:
why do we have this here? We dont want to show the user the status code, but rather the error and the reason